turbinia/workers/__init__.py

```diff
@@ -99,6 +148,16 @@ def add_evidence(self, evidence):
     """
     self.evidence.append(evidence)
 
+  def save_local_file(self, file_):
+    """Saves local file by writing to all non-local output writers.
```

Comment:
This doesn't makes sense to me. Will it save local file by writing to all non-local writers?